docs/opamp-communication-flows/ac-remote-update.md

```diff
@@ -0,0 +1,24 @@
+# Agent Control remote update
+
+Agent Control installation will include the latest Agent Control version. In Kubernetes it will also include Flux 2.15.0.
+These can be updated remotely.
+
+Example message updating Agent Control version:
+
+```yaml
+agents:
+  nr-infra: newrelic/com.newrelic.infrastructure:0.1.0
+chart_version: "1.0.0"
+```
+
+Example message updating Flux version:
+
+```yaml
+agents:
+  nr-infra: newrelic/com.newrelic.infrastructure:0.1.0
+cd_chart_version: "1.0.0"
```

Comment:
Both could be in the same message, I'm not sure if it is worth mentioning.

Reply:
Easy enough. I added it.